calico-enterprise/getting-started/install-on-clusters/kubernetes/helm.mdx

```diff
@@ -38,7 +33,227 @@ In this guide, you install the Tigera Calico operator and custom resource defini
+  
+  --set managedClusters.enabled=true \
+  --set managedClusters.clusters[0].name=$MANAGED_CLUSTER_NAME \
+  --set managedClusters.clusters[0].operatorNamespace=$MANAGED_CLUSTER_OPERATOR_NAMESPACE \
+  --set managedClusters.clusters[0].certificate=$MANAGED_CLUSTER_CERTIFICATE \
+
+  --namespace tigera-operator --create-namespace
+  ```
+
+  or with a LoadBalancer on AWS, add expected annotations to the above installation.
+
+  ```bash
+  --set managementCluster.service.annotations[0].key="service.beta.kubernetes.io/aws-load-balancer-type" \
+  --set managementCluster.service.annotations[0].value=external \
+  --set managementCluster.service.annotations[1].key="service.beta.kubernetes.io/aws-load-balancer-nlb-target-type" \
+  --set managementCluster.service.annotations[1].value=instance \
+  --set managementCluster.service.annotations[2].key="service.beta.kubernetes.io/aws-load-balancer-scheme" \
+  --set managementCluster.service.annotations[2].value=internet-facing \
+  ```
+
+  :::note
+
+  The licenseKey must have `ApiVersion: crd.projectcalico.org/v1`.
+
+  :::
+
+1. You can now monitor progress with the following command:
+
+  ```bash
+  watch kubectl get tigerastatus
+  ```
+
+  Congratulations! You have now installed {{prodname}} for a management cluster using the Helm 3 chart.
+
+</TabItem>
+<TabItem label="Managed" value="Managed">
+
+To install a {{prodname}} [managed](../../../multicluster/create-a-managed-cluster#value) cluster with Helm:
+
+1. [Configure a storage class for Calico Enterprise](../../../operations/logstorage/create-storage).
+
+1. Export the service port number, and the public IP or host of the management cluster. (Ex. "example.com:1234" or "10.0.0.10:1234".)
+
+  ```bash
+  export MANAGEMENT_CLUSTER_ADDR=<your-management-cluster-addr>
+  ```
+
+1. Export the management cluster certificate and managed cluster certificate and key.
+  
+  ```bash
+  export MANAGEMENT_CLUSTER_CRT=<management-cluster-crt>
+  export MANAGED_CLUSTER_CRT=<managed-cluster-crt>
+  export MANAGED_CLUSTER_KEY=<managed-cluster-key>
+  ```
+
+1. Install the Tigera {{prodname}} operator and custom resource definitions using the Helm 3 chart:
+
+  ```bash
+  helm install {{prodnamedash}} projectcalico/tigera-operator --version {{releaseTitle}} -f values.yaml \
+  --set-file imagePullSecrets.tigera-pull-secret=<path/to/pull/secret>,tigera-prometheus-operator.imagePullSecrets.tigera-pull-secret=<path/to/pull/secret> \
+  --set-file licenseKeyContent=<path/to/license/file/yaml> \
+
+  --set logStorage.enabled=false --set manager.enabled=false \
+  
+  --set managementClusterConnection.enabled=true \
+  --set managementClusterConnection.address=$MANAGEMENT_CLUSTER_ADDR \
+  --set managementCluster.management.tls.crt=$MANAGEMENT_CLUSTER_CRT \
+  --set managementCluster.managed.tls.crt=$MANAGED_CLUSTER_CRT \
+  --set managementCluster.managed.tls.key=$MANAGED_CLUSTER_KEY \
+  
+  --namespace tigera-operator --create-namespace
```

Comment:
Is this correct?

Reply:
Does `--create-namespace` work? 

I've successfully tested the behaviour in these docs, and here is a Helm 3 list of install options https://helm.sh/docs/helm/helm_install/#options

Reply:
It must be ok, it just kinda looked funny to me.